coursedata/keywords/fr.yaml

```diff
@@ -6,23 +6,23 @@ turn: tourne
 is: est
 sleep: dors
 add: ajoute
-to_list: jusqua
+to_list: jusqu_a
```

Comment:
Ah now I think I understand your earlier remark about the space versus underscore you meant on this side too.

I reckon this is actually 2 words with a space in between! I think I can make it work so that you can use `jusqu a` cause that will be more authentic I assume. Stay tuned :)

Reply:
After the discussion in #1940 with  @thjazi39 I just realized that the also should be an accent on the "a" since it's  "jusqu'à" for consistency. I haven't seen where this to_list is actually used in the language and so it puzzles me on the actual translation this should have.

Reply:
this 'to' is used when adding something to a list, so it is the `to` in:

`add appel to taarten`

It can potentially be different from the `to` for a range : `for i in range 1 to 5`, for Dutch it is for example (toe versus tot)